.github/workflows/release.yml

```diff
@@ -66,6 +66,11 @@ jobs:
         if: |
           steps.release_exists.outputs.id != ''
 
+      - name: Store SDK version
+        run: |
+          python3 etc/update_api_version.py src/viam/versions.py sdk ${{ env.SDK_VERSION }}
+          echo "Updated SDK version to ${{ env.SDK_VERSION }} in versions.py"
+
```

Comment:
Other alternative that was being done before in case we don't like updating the version on release: getting the SDK version from reading `pyproject.toml` directly from `dial()`.